src/modules/rlm_imap/all.mk.in

```diff
@@ -0,0 +1,49 @@
+#######################################################################
```

Comment:
rlm_imap doesn't need a configure script because it doesn't use libcurl directly, remove `configure`, `configure.ac`, `all.mk.in`.  Copy `all.mk` from `src/modules/rlm_rest/all.mk`, change references to rest to imap.  Remove checks for JSON.